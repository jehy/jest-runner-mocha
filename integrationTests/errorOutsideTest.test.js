'use strict';

const runJest = require('./runJest');

it('Works when it has an error outside the tests', () => {
  return expect(runJest('errorOutsideTest')).resolves.toMatchSnapshot();
});
