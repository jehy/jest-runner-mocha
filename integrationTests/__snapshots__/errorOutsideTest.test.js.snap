// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has an error outside the tests 1`] = `
"[0m[7m[1m[31m FAIL [39m[22m[27m[0m [2mintegrationTests/__fixtures__/errorOutsideTest/__mocha__/__tests__/[22m[1mfile.test.js[22m
  My tests
    [31mâœ•[39m [2mThis test thows an error[22m


    My tests This test thows an error
Error: Some error message
      at mocked-stack-trace

[1mTest Suites:[22m[1m[31m1 failed[39m[22m, 1 total
[1mTests:      [22m[1m[31m1 failed[39m[22m, 1 total
[1mSnapshots:  [22m0 total
[1mTime:[22m        
[2mRan all test suites[22m[2m.[22m
"
`;
